tests/features/custom_page/custom_page.search_field.feature

```diff
@@ -0,0 +1,38 @@
+@api
+Feature: "Custom page" search.
+  In order to use the search
+  As a user of the website
+  I need to be able to view search results in the "Custom page" canonical page.
+
+  Background:
+    Given the following collection:
+      | title  | Code camp  |
+      | logo   | logo.png   |
+      | banner | banner.jpg |
+    And discussion content:
+      | title              | content                        |
+      | Discussion example | This is some dummy discussion. |
+    And "custom_page" content:
+      | title    | collection | body               |
+      | About us | Code camp  | This is an example |
+    # Non UATable step.
+    Given I commit the solr index
+
+    Scenario: View search results in the "Custom page" canonical page.
+      When I am an anonymous user
+      And I go to the homepage of the "Code camp" collection
+      And I click "About us"
```

Comment:
There are two "About us" links on the page. One is in the navigation menu, and one is in a tile in the main content. Which one should I click? Is it intentional this is shown twice on the page?

This seems out of scope for this PR so I made a separate ticket for this: https://webgate.ec.europa.eu/CITnet/jira/browse/ISAICP-2780